sdk/keyvault/azure-security-keyvault-common/inc/azure/keyvault/common/internal/keyvault_pipeline.hpp

```diff
@@ -11,7 +11,7 @@
 
 #include <azure/core/context.hpp>
 #include <azure/core/http/http.hpp>
-#include <azure/core/http/pipeline.hpp>
+#include <azure/core/internal/http/pipeline.hpp>
```

Comment:
Can we try to avoid the internal headers leaking in keyvault public headers since it isn't auto-generated (I know this particular file is internal)?

Reply:
yes